### ISSUE TITLE
False positive when linting om

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#100]
/private/tmp/om/src/main/om/dom.cljs:63:12: error: wrong number of args (1) passed to om.dom/wrap-form-element

**Comment by borkdude** [comment]
When linting omcljs/om using a classpath produces by `lein classpath` I got some weirdness. Turned out I was linting multiple versions of the same namespace.
That was caused because devcards brings in another older version of om:

 ```[devcards "0.2.4" :scope "test" :exclusions [[org.clojure/clojurescript]]]
   [cljsjs/showdown "1.4.2-0" :scope "test"]
   [sablono "0.8.1" :scope "test"]
     [org.omcljs/om "1.0.0-alpha48" :scope "test"]```
Maybe that should be excluded from the classpath in the project.clj?

**Comment by borkdude** [comment]
Maybe emit a warning when we see a ns form with the same name twice? This indicates conflicting namespaces on the classpath.
